packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker/mapper.ts

```diff
@@ -1,16 +1,40 @@
 import { TimeOption, TimeRange, TimeZone, rangeUtil, dateTimeFormat } from '@grafana/data';
+import { formatDateRange } from '@grafana/i18n';
 
+import { getFeatureToggle } from '../../../utils/featureToggle';
 export const mapOptionToTimeRange = (option: TimeOption, timeZone?: TimeZone): TimeRange => {
   return rangeUtil.convertRawToRange({ from: option.from, to: option.to }, timeZone);
 };
 
+const rangeFormatShort: Intl.DateTimeFormatOptions = {
+  dateStyle: 'short',
+  timeStyle: 'short',
+};
+
+const rangeFormatFull: Intl.DateTimeFormatOptions = {
+  dateStyle: 'short',
+  timeStyle: 'medium',
+};
```

Comment:
Rather than manually specifying full date-time componments individually, I think we should prefer to use the style shortcuts that have presets for the specified locale https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#style_shortcuts